flow-defs-libs/react-native.js

```diff
@@ -0,0 +1,7 @@
+declare module 'react-native' {
+  declare module.exports: any
```

Comment:
we gonna need better types here

Reply:
Yes, i just write this types because that there are only two being used. But, i see on some issues, and we'have discussed about typescript. 

Would be a better option, release a new version with the master. Then refactor flow to typescript?

Reply:
> Would be a better option, release a new version with the master. Then refactor flow to typescript?
Yep, I think so.

You can find me at React Brasil Slack or react-native-community Slack to discuss more about that.